osgi.enroute.web.simple.provider/src/osgi/enroute/servlet/api/ConditionalServlet.java

```diff
@@ -0,0 +1,37 @@
+package osgi.enroute.servlet.api;
```

Comment:
You probably want to make this an annotation interface so it can be used as a configuration interface in the DS component. You can then also specify a default value.